src/edge/edgeManager.ts

```diff
@@ -214,20 +214,23 @@ export class EdgeManager {
                 break;
             case Constants.LANGUAGE_NODE:
                 debugCreateOptions = {
-                    ExposedPorts: { "9229/tcp": {}},
-                    HostConfig: {PortBindings: {"9229/tcp": [{HostPort: "9229"}]}}};
+                    ExposedPorts: { "9229/tcp": {} },
+                    HostConfig: { PortBindings: { "9229/tcp": [{ HostPort: "9229" }] } },
+                };
                 break;
             case Constants.LANGUAGE_C:
-                debugCreateOptions = {HostConfig: {Privileged: true}};
+                debugCreateOptions = { HostConfig: { Privileged: true } };
                 break;
             case Constants.LANGUAGE_JAVA:
                 debugCreateOptions = {
-                    HostConfig: {PortBindings: {"5005/tcp": [{HostPort: "5005"}]}}};
+                    HostConfig: { PortBindings: { "5005/tcp": [{ HostPort: "5005" }] } },
+                };
                 break;
             case Constants.LANGUAGE_PYTHON:
                 debugCreateOptions = {
-                    ExposedPorts: {"5678/tcp": {}},
-                    HostConfig: {PortBindings: {"5678/tcp": [{HostPort: "5678"}]}}};
+                    ExposedPorts: { "5678/tcp": {} },
+                    HostConfig: { PortBindings: { "5678/tcp": [{ HostPort: "5678" }] } },

```

Comment:
actually I will prefer no space between curly braces.

Reply:
This is the result of a "Format code" command...